tests/TestCase/View/Helper/MigrationHelperTest.php

```diff
@@ -262,22 +262,22 @@ public function testColumn()
      */
     public function testValue()
     {
-        $this->assertEquals('null', $this->helper->value(null));
-        $this->assertEquals('null', $this->helper->value('null'));
-        $this->assertEquals('true', $this->helper->value(true));
-        $this->assertEquals('false', $this->helper->value(false));
-        $this->assertEquals(1, $this->helper->value(1));
-        $this->assertEquals(-1, $this->helper->value(-1));
-        $this->assertEquals(1.5, $this->helper->value(1.5));
-        $this->assertEquals(1.5, $this->helper->value('1.5'));
-        $this->assertEquals(1, $this->helper->value('1'));
+        $this->assertSame('null', $this->helper->value(null));
+        $this->assertSame('null', $this->helper->value('null'));
+        $this->assertSame('true', $this->helper->value(true));
+        $this->assertSame('false', $this->helper->value(false));
+        $this->assertSame(1, $this->helper->value(1));
```

Comment:
This test [is failing](https://travis-ci.com/github/cakephp/migrations/jobs/399765313#L435)

Reply:
@markstory, thanks for your review.

Let me fix them :).

Reply:
Failed assertion has been fixed now.